### ISSUE TITLE
The performance of pre-trained model on ImageNet?

### ISSUE THREAD

**Issue by ycszen** [CSAILVision/semantic-segmentation-pytorch#104]
Could you offer the accurate accuracy or error rate on the ImageNet of the pre-trained model?
In your code of resnet, the first 7x7 conv is replaced by three 3x3 convs. This will improve the performance obviously. Therefore, I want to know the accurate accuracy of your pre-trained resnet model to make some ablation study. 
Thank you.

**Comment by hangzhaomit** [comment]
You can directly use our pretrained model to evaluate performance on ImageNet.
